src/authenticator.rs

```diff
@@ -1070,17 +1069,22 @@ where
             Some(ATTEMPT_COUNTER_DEFAULT_RETRIES),
             true
         ))
-        .map_err(|_| iso7816::Status::UnspecifiedNonpersistentExecutionError)?;
+        .map_err(|e| Self::_debug_trussed_backend_error(e, line!()))?;
         Ok(())
     }
 
+    fn _debug_trussed_backend_error(e: trussed::Error, l: u32) -> iso7816::Status {
```

Comment:
Why the `_` ?

Reply:
I did not want to clutter Authenticator namespace further - this will be extracted in the next PRs (all prefixed with `_`).